enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -803,6 +803,16 @@ bool ActivityAnalyzer::isConstantInstruction(TypeResults const &TR,
       InsertConstantInstruction(TR, I);
       return true;
     }
+#if defined(ENZYME_IFX) && (LLVM_VERSION_MAJOR == 15)
```

Comment:
I don't think this needs either the version nor IFX limitation hhere

Reply:
Cool, yes I'll remove that. I was just using ENZYME_IFX as a feature flag. And was being cautious with the llvm version in case the intrinsic changes in a future version of IFX